.travis.yml

```diff
@@ -5,10 +5,16 @@ go:
 
 services:
   - docker
+  - postgresql
 
 sudo: false
 
+before_script:
+  - psql -c 'create database docker_test;' -U postgres
+
 install:
+  - go get -u github.com/jteeuwen/go-bindata/...
+
   # not using gb atm as it does not have support for static linking
   #- go get github.com/constabulary/gb/...
   - make build-linux
```

Comment:
that target in the makefile builds a statically linked executable, this just means that the executable has everything it needs to run and does not rely on linking to any os stuff (or other libraries).  normally when you compile something it will create a dynamically linked executable which will go outside of itself to use anything which is outside of our application.

this allows us to use the `FROM scratch` in our docker file which means that there is no base  OS this keeps the docker file very small, means it is more secure and prevents us from having to worry about os updates and the like.